### ISSUE TITLE
Object.activate is deprecated.

### ISSUE THREAD

**Issue by none-da** [AtomLinter/linter-pylama#9]
AtomLinter v0.X.Y API has been deprecated. Please refer to the Linter docs to update and the latest API: https://github.com/AtomLinter/Linter/wiki/Migrating-to-the-new-API
```
Object.activate (/home/jnandakishore/.atom/packages/linter/lib/main.coffee:57:8)
Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:240:19)
<unknown> (/usr/share/atom/resources/app.asar/src/package.js:221:30)
Package.measure (/usr/share/atom/resources/app.asar/src/package.js:165:15)
Package.activate (/usr/share/atom/resources/app.asar/src/package.js:213:14)
PackageManager.activatePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:452:21)
```

**Comment by gmist** [comment]
It has been fixed in v0.1.2 adc96d80c88f5f3700cf24206db64caf5ccb71ea You've already upgraded to the latest version?

**Comment by gmist** [comment]
It seems that the problem is solved. If it is not - just open this issue again.
